docs/010-welcome-to-civicactions/team-resume-instructions.md

```diff
@@ -4,9 +4,8 @@ We sometimes get requests for team resumes. We use a template for formatting tea
 
 ## Process
 
-1. Look for an email from your onboarder introducing you to our Sales team asking that you schedule a 30 minute meeting to work on your resume and get an overview of our Sales/BD process.
-2. The Sales team will make a copy of the resume template and then connect with you to provide a link to your CivicActions resumes and help you complete the resume as needed.
-3. The Sales team will connect with you as needed to complete it. They may schedule a quick Zoom meeting to complete it together as well.
-4. The Sales team will also request copies of your degrees and certificates to keep on file.
-5. When you are complete with your resume, email the Sales team a link back to the same resume draft first provided, and also include the certs and degrees as noted above.
-6. The Sales team will do final review and save your resume in the team resumes folder.
+1. Make a copy of the resume template from here: https://docs.google.com/document/d/1jVjUOql0sogCaIWGClpRpcUDir62SKT7HJ8r0rvtZkY/edit
+2. Follow the examples to fill out your information.
+3. When you are complete with your resume, export it as a PDF and save it in the resumes folder on Google Drive here: https://drive.google.com/drive/folders/1Wt1KObSnraOnyDjUtK9chSHldsis9hiC.
```

Comment:
I'm not sure if this is the correct location if we're looking for general drive directory to store resumes. This looks like it's project-specific?